service_capacity_modeling/hardware/profiles/pricing/aws/3yr-reserved.json

```diff
@@ -51,6 +53,31 @@
       "m6id.2xlarge":  {"annual_cost": 1563.32},
       "m6id.4xlarge":  {"annual_cost": 3126.64},
       "m6id.8xlarge":  {"annual_cost": 6253.28},
+
```

Comment:
No longer needed after the pricing Pr is merged (#102), but I think we are on the same page here.